fdbclient/MultiVersionTransaction.actor.cpp

```diff
@@ -1784,16 +1784,15 @@ void MultiVersionApi::setNetworkOptionInternal(FDBNetworkOptions::Option option,
 	} else if (option == FDBNetworkOptions::CLIENT_THREADS_PER_VERSION) {
 		MutexHolder holder(lock);
 		validateOption(value, true, false, false);
-		ASSERT(!networkStartSetup);
+		if (networkStartSetup) {
+			throw invalid_option();
+		}
```

Comment:
I see there are similar asserts for other options, we can change them to throwing exception too

Reply:
`SUPPORTED_CLIENT_VERSIONS` and `EXTERNAL_CLIENT` are both somewhat hidden internal options that aren't meant to be called outside of the MVC, which is why they have asserts.